ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/operators/SelectOperator.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.physical.pull.operators;
+
+import io.confluent.ksql.planner.plan.FilterNode;
+import java.util.Objects;
+
+public class SelectOperator extends AbstractPhysicalOperator implements UnaryPhysicalOperator {
+
+  private final FilterNode logicalNode;
+  private AbstractPhysicalOperator child;
+
+  public SelectOperator(final FilterNode logicalNode) {
+    this.logicalNode = Objects.requireNonNull(logicalNode);
+  }
+
+  @Override
+  public void open() {
+    child.open();
+  }
+
+  @Override
+  public Object next() {
+    return child.next();
```

Comment:
I assume this doesn't currently do anything because this is done at the rocksdb layer?

Reply:
This doesn't do anything because currently we only support single key equality and nothing else in the WHERE clause. When we make it more general and allow others stuff as well, (like functions) we will need to add logic here.